crates/sdk/src/default_main.rs

```diff
@@ -84,6 +84,8 @@ struct TestCommand {
     configuration: PathBuf,
     #[arg(long, value_name = "DIRECTORY", env = "HASURA_SNAPSHOTS_DIR")]
     snapshots_dir: Option<PathBuf>,
+    #[arg(long, help = "Turn off validations for query responses")]
+    no_validate_responses: bool,
```

Comment:
this flag copied from the [ndc-test implementation](https://github.com/hasura/ndc-spec/pull/141/commits/b6f01f54ee254e277effcfffa42817660c1f7251)